server/src/main/java/org/elasticsearch/env/NodeEnvironment.java

```diff
@@ -698,11 +701,13 @@ protected void closeInternal() {
          */
         private final Semaphore mutex = new Semaphore(1);
         private int waitCount = 1; // guarded by shardLocks
+        private String lockDetails;
         private final ShardId shardId;
 
         InternalShardLock(ShardId shardId) {
             this.shardId = shardId;
             mutex.acquireUninterruptibly();
+            lockDetails = "initial creation lock";
```

Comment:
I am not a native speaker but that message sounds a bit odd to me. Maybe `initial creation of lock`?

Reply:
It's technically okay because 'lock' in my version is being used as a verb, but to clarify I'll change it so it's a noun :)